classes/label.rb

```diff
@@ -0,0 +1,18 @@
+require 'securerandom'
+
+class Label
+  attr_accessor :title, :color
+  attr_reader :id, :items
+
+  def initialize(title, id = SecureRandom.random_number(1000))
+    @id = id
+    @title = title
+    @color = color
```

Comment:
- [ ] Kindly, add the color variable to the parameters that must be provided to create a new instance of Label. You have defined the instance variable, just add it to the initialize parenthesis 😉

Reply:
OMG, I didn't see that, I could swear it was there! Thanks!

Reply:
Changes appplied!